Dashboard/Client/SessionManagement/ClientSessionManager.cs

```diff
@@ -29,9 +32,10 @@ public ClientSessionManager()
             _serializer = new Serializer();
             _communicator = CommunicationFactory.GetCommunicator();
             _communicator.Subscribe(moduleIdentifier, this);
+            _contentClient = ContentClientFactory.getInstance();
+
             TraceManager session = new();
-            session.TraceListener();
-           
+            session.TraceListener(); 

```

Comment:
Add trace before initializing other modules. and also initialize whiteboard and screen share.

Reply:
Initialisations on client side?

Reply:
yup